Tests/Configuration/pageObject.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you please use separate page object models for each page.  We have created some example for you in this template project: https://github.com/UKHO/playwright-template

A good example of a page object model is https://github.com/UKHO/playwright-template/blob/main/tests/page-object-model/formpage.ts

Reply:
resolved and remove the pageobject json file, already we are following page object model for the individual pages